test/functional/plugin/lsp/incremental_sync_spec.lua

```diff
@@ -265,12 +460,12 @@ describe('incremental synchronization', function()
               line = 0
             },
             ['end'] = {
-              character = 17,
+              character = 12,
               line = 0
             }
           },
-          rangeLength = 6,
-          text = '\ntest3'
```

Comment:
This test change looks problematic

Reply:
How so? It seems we're just replacing a space with a newline here.